remixer_core/src/main/java/com/google/android/libraries/remixer/Remixer.java

```diff
@@ -16,13 +16,25 @@
 
 package com.google.android.libraries.remixer;
 
+import com.google.android.libraries.remixer.sync.SynchronizationMechanism;
 import java.util.ArrayList;
 import java.util.HashMap;
 import java.util.List;
 import java.util.Locale;
 
 /**
- * Contains a list of {@link Variable}es.
+ * Contains a list of {@link Variable}s or {@link Trigger}s.
+ *
+ * <p>The Remixer object is the heart and soul of Remixer as a framework, it coordinates syncing and
+ * value changes.
+ *
+ * <p>For value and data syncing/persistence (both locally across activities, and persisting/saving
+ * to cloud backends, etc) and keeping a global state, the Remixer object relies on a {@link
+ * SynchronizationMechanism} set at initialization time (usually in your Application.onCreate()).
+ *
+ * <p>If you do not set a SynchronizationMechanism, remixer will not try to sync values across
+ * different contexts (activities) or check for consistency among those. It is recommended you
+ * always use one (even if it is just the LocalValueSyncing one) so you catch consistency errors.
```

Comment:
So this makes me wonder. Why doesn't the Remixer object just always initialize 
```
this. synchronizationMechanism = new LocalValueSyncing();
```

This way, clients don't have to be bothered to initialize in the 99% case ("It is recommended you always use one") and you get consistency checks for free. If clients do want to set their own synchronizationMechanism, they can easily do so in Application.onCreate()